homeassistant/components/advantage_air/climate.py

```diff
@@ -185,15 +158,16 @@ async def async_set_temperature(self, **kwargs):
 class AdvantageAirZone(AdvantageAirClimateEntity):
     """AdvantageAir Zone control."""
 
-    @property
-    def name(self):
-        """Return the name."""
-        return self._zone["name"]
+    _attr_hvac_modes = ZONE_HVAC_MODES
+    _attr_supported_features = SUPPORT_TARGET_TEMPERATURE
 
-    @property
-    def unique_id(self):
-        """Return a unique id."""
-        return f'{self.coordinator.data["system"]["rid"]}-{self.ac_key}-{self.zone_key}'
+    def __init__(self, instance, ac_key, zone_key):
+        """Initialize an AdvantageAir Zone control."""
+        super().__init__(instance, ac_key, zone_key)
+        self._attr_name = self._zone["name"]
```

Comment:
This appears to prevent the name from being updated by the source without a restart.